wagtail_wordpress_import/test/tests/test_shortcode_handlers.py

```diff
@@ -304,3 +326,136 @@ def test_construct_block_method_output(self):
         )
         self.assertEqual(json["value"]["alignment"], "center")
         self.assertEqual(json["value"]["link"], "https//www.example.com/bar/")
+
+
+class TestCaptionHandler(TestCase):
+    @responses.activate
+    def test_absence_of_image(self):
+        handler = CaptionHandler()
+        responses.add(
+            responses.GET,
+            "https://www.example.com/images/foo.jpg",
+            body=mock_image(),
+            status=200,
+            content_type="image/jpeg",
+        )
+        wagtail_custom_html = """
+        <wagtail_block_caption id="attachment_46162" align="aligncenter" width="600">
+            <a href="http://www.example.com/">
+                
+            </a>This is a caption about the image
+        </wagtail_block_caption>"""
```

Comment:
**Question**: We're testing that if the image is not found at the URL, there's no image in the imported document? Why add the response at all?

Reply:
This is resolved in 150afdd.